util/util.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Difference returns the difference between two arrays
+func Difference(arr1, arr2 []int64) (result []int64) {
```

Comment:
Why does the second argument is int64[] and the first has no declaration?

Reply:
I learned a new trick :)
you can specify the type only once if all arguments are of the same type